server/graphql/v1/TransactionInterface.js

```diff
@@ -49,7 +49,6 @@ export const TransactionInterfaceType = new GraphQLInterfaceType({
       collective: { type: CollectiveInterfaceType },
       type: { type: GraphQLString },
       description: { type: GraphQLString },
-      privateMessage: { type: GraphQLString },
```

Comment:
Why remove `privateMessage` here?

Reply:
@znarf I did it because `privateMessage` is loaded either from `Order` or `Expense` it's not loaded from the `Transaction` itself, should I return it back?

Reply:
It's all right.